angular-client/src/enumeration/ImportantDataType.ts

```diff
@@ -0,0 +1,5 @@
+export enum ImportantDataType {
```

Comment:
is this a folder you made? enumeration seems kinda specific for an overarching folder name

Reply:
also I feel like we could do better for a name than 'ImportantDataType' unless that's how you've been refering to it in scylla

Reply:
Yeah i didnt really know what to name it, the folder is for all enumerations i add , should probably be called enumerations not enumeration. DataType is what its called in scylla, but its like data types that were specifically referencing rather than just general plotting so idk